components/Column.js

```diff
@@ -77,30 +77,56 @@ export default class Column extends Component {
   }
 
   // Renders the "bricks" within the columns
-  // _renderBricks :: [images] -> [TouchableTag || ImageTag...]
-  _renderBricks (bricks) {
-    return bricks.map((brick, index) => {
-      const gutter = (index === 0) ? 0 : brick.gutter;
-      const key = `RN-MASONRY-BRICK-${brick.column}-${index}`;
+  // _renderBrick :: images -> [TouchableTag || ImageTag...]
+  _renderBrick (data) {
+      // Example Data Structure
+      // {
+      //   "item": {
+      //     "uri": "https://img.buzzfeed.com/buzzfeed-static/static/2016-01/14/20/campaign_images/webdr15/which-delicious-mexican-food-item-are-you-based-o-2-20324-1452822970-1_dblbig.jpg",
+      //     "column": 0,
+      //     "dimensions": {
+      //       "width": 625,
+      //       "height": 415
+      //     },
+      //     "width": 180.675,
+      //     "height": 119.96820000000001,
+      //     "gutter": 3.65
+      //   },
+      //   "index": 9
+      // }
+      const brick = data.item;
+      const gutter = (data.index === 0) ? 0 : brick.gutter;
```

Comment:
Will the change I just made to have [bricks be sorted](https://github.com/brh55/react-native-masonry/blob/e46fea5bba027011dc3c54321a2a0dfefbe64fe8/components/Masonry.js#L16) interfere with this `index`. I also set that in `Masonry.js`